LEGO1/legoanimationmanager.h

```diff
@@ -23,10 +23,11 @@ class LegoAnimationManager : public MxCore {
 	// FUNCTION: LEGO1 0x1005ec90
 	inline virtual MxBool IsA(const char* p_name) const override // vtable+0x10
 	{
-		return !strcmp(p_name, ClassName()) || MxCore::IsA(p_name);
+		return !strcmp(p_name, LegoAnimationManager::ClassName()) || MxCore::IsA(p_name);
```

Comment:
In this it actually *does* require the virtual function call to `ClassName`. Maybe this is a bug by the developers but it's how it's done.